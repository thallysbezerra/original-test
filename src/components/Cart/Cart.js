import React from 'react';
import PropTypes from 'prop-types';

import './Cart.scss';

const Cart = ({ itemsQuantity }) => (
	<div className="cart">
		<i className="cart__icon fas fa-shopping-bag" />
		<span className="car__items-quantity">{itemsQuantity}</span>
	</div>
);

Cart.propTypes = {
	itemsQuantity: PropTypes.number.isRequired
};

Cart.defaultProps = {
	itemsQuantity: 0
};

export default Cart;
