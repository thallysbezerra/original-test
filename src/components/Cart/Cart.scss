@import '../../css/Theme.scss';

.cart {
	color: $grey-seven;

	&__icon {
		font-size: 21px;
		margin-right: 5px;
	}
}
