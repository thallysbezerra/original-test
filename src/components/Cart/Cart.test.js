import React from 'react';
import Cart from './Cart';

import renderer from 'react-test-renderer';

describe('Cart UI', () => {
	it('<Cart /> must have the snapshot version.', () => {
		const tree = renderer.create(<Cart />);

		expect(tree.toJSON()).toMatchSnapshot();
	});
});
