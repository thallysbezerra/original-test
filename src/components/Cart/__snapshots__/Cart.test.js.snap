// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart UI <Cart /> must have the snapshot version. 1`] = `
<div
  className="cart"
>
  <i
    className="cart__icon fas fa-shopping-bag"
  />
  <span
    className="car__items-quantity"
  >
    0
  </span>
</div>
`;
